src/toxcore/messenger/packet/file_control.rs

```diff
@@ -16,6 +16,17 @@ pub enum TransferDirection {
     Receive
 }
 
+impl TransferDirection {
+    /// Toggle direction
+    pub fn toggle(&self) -> Self {
```

Comment:
It's `Copy` so can be taken by value. btw, doesn't clippy warn here?

Reply:
Now, there are much warnings when compile. So I didn't check warnings.
All warnings will be removed.